osbrain/tests/test_agent.py

```diff
@@ -91,6 +91,21 @@ def run(self):
     assert a0.ping() == 'pong'
 
 
+def test_agent_configure(nsproxy):
```

Comment:
Maybe a short docstring for the test and you are good to go! :blush:

(Even if it is a simple "Test agent's `configure()` method, which should be able to start timers.")